cfn-resources/global-cluster-config/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: build test clean
+
+build:
+	make -f makebuild  # this runs build steps required by the cfn cli
+
+test:
+	cfn generate
+	env GOOS=linux go build -ldflags="-s -w" -o bin/handler cmd/main.go
+
+clean:
+	rm -rf bin
```

Comment:
@rdharmedran you didnt have any issues with this makefile? i think you are missing some things can you check the other resources?h